src/corehost/packaging/Microsoft.NETCore.App/project.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160321-04",
+    "Microsoft.CSharp": "4.0.1-rc2-23931",
+    "Microsoft.NETCore.DotNetHostPolicy": "1.0.1-beta-002200",
```

Comment:
If you see the message in my commit the my current versioning model for the meta-package is "1.0.0" + "NetStandardLibrary.BuildNumber" + "CLI build Number" See https://github.com/dotnet/cli/pull/2274/files#diff-a4a50058597e5e9e728f0ac284a74259R47